actors/builtin/miner/miner_state.go

```diff
@@ -227,8 +227,9 @@ func (st *State) ComputeProvingSet(store adt.Store) ([]abi.SectorInfo, error) {
 		// if not a temp fault sector, add to computed proving set
 		if !fault {
 			sectorInfos = append(sectorInfos, abi.SectorInfo{
-				SealedCID:    ssinfo.Info.SealedCID,
-				SectorNumber: ssinfo.Info.SectorNumber,
+				SealedCID:       ssinfo.Info.SealedCID,
+				SectorNumber:    ssinfo.Info.SectorNumber,
+				RegisteredProof: ssinfo.Info.RegisteredProof,
```

Comment:
Remember to map the seal proof to the post proof.

Perhaps rename SectorInfo to PoStSectorInfo to clarify.

Reply:
@anorth 

> Remember to map the seal proof to the post proof.

I was going to do the mapping in the implementation of `VerifyPoSt` so as to keep this diff small. Other structs (e.g. `PoStCandidate`) embed `RegisteredProof`, so for consistency's sake I did the same with `SectorInfo`. Thoughts?

Reply:
Ok, great. Could you simply document that in the struct?

Reply:
done